lightly_studio_view/e2e/general/bussines-flow-1.e2e-test.ts

```diff
@@ -91,52 +92,21 @@ test.describe('bussines-flow1', () => {
 
         // Create the tag with unique name
         await samplesPage.createTag(catsTagName);
-        await samplesPage.goto();
-
         expect(samplesPage.getTagsMenuItem(catsTagName)).toBeDefined();
 
+        // Filter by the newly created tag
         await samplesPage.pressTag(catsTagName);
         expect(await samplesPage.getSamples().count()).toBe(1);
-    });
-    test('Remove cat text filter and check UI', async ({ page, samplesPage }) => {
```

Comment:
This test split was removed.

The reason is that it uses the `samplesPage` fixture, which in the setup calls `samplesPage.goto`. Therefore many of the actions in the test did not make sense. E.g. clearing the search - it was already cleared by the goto.